vcrun6sp6 crash

When attempting to remove `stdole2.dll` file the verb crashes
```
Extracting to: /home/jonasz/.Phoenicis/containers//wineprefix//Shogo//drive_c/vcrun6sp6/
Extracting cabinet: /home/jonasz/.Phoenicis/resources///Vs6sp6.exe
  extracting vcredist.exe

All done, no errors.
[ERROR] org.phoenicis.multithreading.ControlledThreadPoolExecutorService (l.64) - Path "/home/jonasz/.Phoenicis/containers//wineprefix//Shogo//drive_c/windows/system32/stdole2.dll" does not exist
	at org.phoenicis.tools.files.FileUtilities.remove(FileUtilities.java:165)
	at <js> remove(Unnamed:101:2143-2172)
	at <js> go(Unnamed:38:1348-1389)
	at <js> install(Unnamed:52:1766-1789)
	at org.graalvm.polyglot.Value.invokeMember(Value.java:459)
	at org.phoenicis.engines.VerbsManager.lambda$installVerb$0(VerbsManager.java:71)
	at org.phoenicis.scripts.session.PhoenicisInteractiveScriptSession.eval(PhoenicisInteractiveScriptSession.java:35)
	at org.phoenicis.scripts.interpreter.BackgroundScriptInterpreter.lambda$createInteractiveSession$1(BackgroundScriptInterpreter.java:45)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by host exception: java.lang.IllegalArgumentException: Path "/home/jonasz/.Phoenicis/containers//wineprefix//Shogo//drive_c/windows/system32/stdole2.dll" does not exist

[WARNING] 
org.graalvm.polyglot.PolyglotException: Path "/home/jonasz/.Phoenicis/containers//wineprefix//Shogo//drive_c/windows/system32/stdole2.dll" does not exist
    at org.phoenicis.tools.files.FileUtilities.remove (FileUtilities.java:165)
    at <js>.remove (Unnamed:101)
    at <js>.go (Unnamed:38)
```
I've looked and there really is no such file.

--- a/Engines/Wine/Verbs/vcrun6sp6/script.js
+++ b/Engines/Wine/Verbs/vcrun6sp6/script.js
@@ -35,7 +35,7 @@ class Vcrun6SP6 {
         remove(`${system32directory}/msvcrt.dll`);
         remove(`${system32directory}/oleaut32.dll`);
         remove(`${system32directory}/olepro32.dll`);
-        remove(`${system32directory}/stdole2.dll`);
+        remove(`${system32directory}/stdole2.tlb`);
 
         wizard.wait(tr("Please wait while {0} is installed...", "vcrun6sp6"));
 

